doc/docs.md

```diff
@@ -310,6 +311,17 @@ fn main() {
 Unlike most languages, variable shadowing is not allowed. Declaring a variable with a name
 that is already used in a parent scope will cause a compilation error.
 
+An exception to this rule is shadowing imported module names:
```

Comment:
I wouldn't call it an exception, because this is not variable shadowing.

Reply:
done I think